spec/rubocop/cop/rspec/describe_class_spec.rb

```diff
@@ -51,6 +51,21 @@
     RUBY
   end
 
+  it 'ignores non-class constants' do
+    expect_no_offenses(<<-RUBY)
+      describe 'Some::VERSION' do
+      end
+    RUBY
+  end
+
+  it 'ignores assignment of non-class constants' do
+    expect_no_offenses(<<-RUBY)
+      describe(const_name = 'Some::VERSION') do
+        subject { Object.const_get(const_name) }
```

Comment:
I've never seen this notation before you've mentioned it in the original ticket.
WDYT of:
```ruby
describe 'Some::VERSION' do
  subject { Object.const_get(self.class.description) }
```
We could get rid of the complex dance with `lvasgn`, and you'll get your case covered, and the default one for string description as well.

Reply:
I think it'a better approach. Thank you!